core/bentley/src/BentleyError.ts

```diff
@@ -277,21 +276,21 @@ export enum IModelHubStatus {
   EventTypeDoesNotExist = IMODELHUBERROR_BASE + 32,
   EventSubscriptionDoesNotExist = IMODELHUBERROR_BASE + 33,
   EventSubscriptionAlreadyExists = IMODELHUBERROR_BASE + 34,
-  ProjectIdIsNotSpecified = IMODELHUBERROR_BASE + 35,
-  FailedToGetProjectPermissions = IMODELHUBERROR_BASE + 36,
-  FailedToGetProjectMembers = IMODELHUBERROR_BASE + 37,
+  ITwinIdIsNotSpecified = IMODELHUBERROR_BASE + 35,
```

Comment:
These names are used when parsing iModelHub errors. Since errors are not renamed in iModelHub, this library won't be able to parse errors correctly.

Reply:
Is the error not sent as the error code?